config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -221,6 +221,45 @@ postsubmits:
     rerun_auth_config:
       github_users:
       - alvaroaleman
+  - name: post-test-infra-push-prow-staging
+    cluster: test-infra-trusted
+    # Runs on more than just the Prow dir to include some additional images that we publish to gcr.io/k8s-prow.
+    run_if_changed: '^(hack/make-rules|prow|ghproxy|label_sync/.+\.go|robots/commenter|robots/pr-creator|robots/issue-creator|testgrid/cmd|gcsweb)'
```

Comment:
We're still including a bunch of directories for images that are not pushed by this job, right?
Are you planning to continue publishing these images with the core Prow images?